hutch_python/env_version.py

```diff
@@ -0,0 +1,73 @@
+"""
+Utilities for getting relevant environment information.
+"""
+from __future__ import annotations
+
+import logging
+import os
+import os.path
+
+import pkg_resources
+
+logger = logging.getLogger(__name__)
+
+_dev_ignore_list = ['x86_64-rhel7-opt', 'x86_64-linux-opt']
```

Comment:
If the goal is to cut out the daq lib spam, there a few more architectures to pick here
```
$ ls /reg/g/pcds/dist/pds/current/build/pdsapp/lib/
i386-linux-dbg  i386-linux-opt  x86_64-linux  x86_64-linux-dbg  x86_64-linux-opt  x86_64-rhel7  x86_64-rhel7-dbg  x86_64-rhel7-opt
```
or we could also switch the targets to `pdsapp` and `ami`, maybe? To specifically exclude the control and monitoring softwares respectively?

Reply:
I didn't know the proper targets here, good to know.